source/extensions/filters/http/file_system_buffer/filter.cc

```diff
@@ -0,0 +1,19 @@
+#include "source/extensions/filters/http/file_system_buffer/filter.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace FileSystemBuffer {
+
+FileSystemBufferFilter::FileSystemBufferFilter(
+    std::shared_ptr<FileSystemBufferFilterConfig> base_config)
+    : base_config_(base_config) {}
+
+const std::string& FileSystemBufferFilter::filterName() {
+  CONSTRUCT_ON_FIRST_USE(std::string, "dropbox.envoy.file_system_buffer");
```

Comment:
I assume copy/paste issue from internal code. Can you update to the OS typical naming?

Reply:
Oops! Done.